StyleCop.Analyzers/StyleCop.Analyzers.Test/DocumentationRules/SA1642UnitTests.cs

```diff
@@ -939,8 +939,7 @@ public ClassName()
 {{
     public class ClassName
     {{
-        /// <summary>
-        /// Initializes a new instance of the <see cref=""ClassName""/> class.</summary>
+        /// <summary>Initializes a new instance of the <see cref=""ClassName""/> class.</summary>
```

Comment:
This test began to fail after I made my changes. In the issue it said this should only apply when changing the reference in a summary. However, before I implement that part I wanted to make sure I wasn't taking it too literal. It seems to me if I start with a single line comment I want it to stay on a single line. Or at the very least have each part on a separate line like below.

```
/// <summary>
/// Initializes a new instance of the <see cref=""TestClass""/> class.
/// </summary>
```

Reply:
I'm fine with this change in behavior. The test `TestConstructorEmptyDocumentationAsync` below covers the case of empty multi-line comments.

Reply:
This testcase contains the exact issue you're trying to solve, so it would have been bad if it hadn't failed 😉